specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations-authoring.json

```diff
@@ -0,0 +1,2703 @@
+          "description": "Contains the data related to intent evaluation.",
+          "$ref": "#/definitions/EvaluationResultIntentObject"
+        }
+      }
+    },
+    "EvaluationResultEntityObject": {
+      "required": [
+        "confusionMatrix",
+        "entities",
+        "microF1",
+        "microPrecision",
+        "microRecall",
+        "macroF1",
+        "macroPrecision",
+        "macroRecall"
+      ],
+      "type": "object",
+      "properties": {
+        "confusionMatrix": {
+          "description": "Represents the confusion matrix between two entities (the two entities can be the same entity)\n            the matrix is between the actual entity that was labelled and the entity that was predicted.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "object",
+            "additionalProperties": {
+              "$ref": "#/definitions/ConfusionMatrixObject"
+            }
+          }
+        },
+        "entities": {
+          "description": "Represents the entities evaluation",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/EvaluationResultIntentAndEntityObject"
+          }
+        },
+        "microF1": {
+          "format": "float",
+          "description": "Represents the micro F1",
+          "type": "number"
+        },
+        "microPrecision": {
+          "format": "float",
+          "description": "Represents the micro precision",
+          "type": "number"
+        },
+        "microRecall": {
+          "format": "float",
+          "description": "Represents the micro recall",
+          "type": "number"
+        },
+        "macroF1": {
+          "format": "float",
+          "description": "Represents the macro F1",
+          "type": "number"
+        },
+        "macroPrecision": {
+          "format": "float",
+          "description": "Represents the macro precision",
+          "type": "number"
+        },
+        "macroRecall": {
+          "format": "float",
+          "description": "Represents the macro recall",
+          "type": "number"
+        }
+      }
+    },
+    "ConfusionMatrixObject": {
+      "required": [
+        "normalizedValue",
+        "rawValue"
+      ],
+      "type": "object",
+      "properties": {
+        "normalizedValue": {
+          "format": "float",
+          "description": "Represents normalized value in percentages.",
+          "type": "number"
+        },
+        "rawValue": {
+          "format": "float",
+          "description": "Represents raw value.",
+          "type": "number"
+        }
+      }
+    },
+    "EvaluationResultIntentAndEntityObject": {
+      "required": [
+        "f1",
+        "precision",
+        "recall",
+        "truePositivesCount",
+        "trueNegativesCount",
+        "falsePositivesCount",
+        "falseNegativesCount"
+      ],
+      "type": "object",
+      "properties": {
+        "f1": {
```

Comment:
Same comment for all these floats as well. What is the range callers should expect in request and/or response objects?